src/com/xilinx/rapidwright/edif/EDIFCellInst.java

```diff
@@ -88,57 +88,66 @@ public EDIFName getViewref() {
     public void setViewref(EDIFName viewref) {
         this.viewref = viewref;
     }
-
+    
     /**
-     * This gets a map of all the port refs on the cell instance.  
-     * @return A map of port ref names to port ref objects.
+     * Creates a new map of all the EDIFPortInst objects stored on this EDIFCellInst.  The new map
+     * contains a copy of EDIFPortInsts available at the time of invocation as returned from 
+     * {@link #getPortInstList()}.      
+     * @return A map of EDIFPortInst names ({@link EDIFPortInst#getName()} to the corresponding objects.
```

Comment:
```suggestion
     * @return A map of EDIFPortInst names ({@link EDIFPortInst#getName()} to the corresponding objects.
     * @deprecated
```

Reply:
Is this change tagging it to be `@deprecated`?  It seems like it already is.

Reply:
My mistake, sorry!